src/components/myPage/InfoSection.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from '@emotion/styled';
+
+interface UserInfoList {
+  title: string;
+  count: number;
+}
+
+export default function InfoSection() {
+  const tempUserInfoList: UserInfoList[] = [
+    { title: '게시물', count: 10 },
+    { title: '팔로워', count: 601 },
+    { title: '팔로우', count: 554 },
+  ];
+
+  return (
+    <StUserInfoListWrapper>
+      {tempUserInfoList.map((userInfo, idx) => {
+        return (
+          <StUserInfoList key={`${userInfo.title}-${idx}`}>
+            <StTextWrapper>
+              {userInfo.title}
+              <StUserInfoCount>{userInfo.count}</StUserInfoCount>
+            </StTextWrapper>
+          </StUserInfoList>
```

Comment:
이런 구조는 보통 아래와 같이 짜기 쉬운데 혹시 이렇게 한 특별한 이유가 있을까요? css 봐도 잘 모르겠는데 특별한 이유가 없다면 아래 형태가 더 가독성 좋지 않나 싶습니다!
```suggestion
          <StUserInfo key={`${userInfo.title}-${idx}`}>
            <StUserInfoTitle>{userInfo.title}</StUserInfoTitle>
            <StUserInfoCount>{userInfo.count}</StUserInfoCount>
          </StUserInfo>
```

Reply:
맞아요! 인스타그램에서 작성되어있던 태그구조를 따와서 구현하였씁니다!